### PR TITLE
Motor speed invertedFactor usage

### PR DESCRIPTION
**Pull request by THEb0nny**
Added use of the invertedFactor() method in the speed() method. Done in the same way as for the case of the angle() method.

Before change
Screenshot_3

After the change
Screenshot_4

But the simulator itself still uses getMotorData(), which does not take into account motor inversion. Therefore, the value of the speed or angle with a minus is displayed there. I could not add invertedFactor() to getMotorData() for a specific motor, because the simulator immediately starts displaying motors that are not used in the program.

Changes like this...
https://github.com/microsoft/pxt-ev3/pull/999

### CODE REVIEW

**Comment by THEb0nny** [comment]
@jwunderl please see changes...

File: libs/core/output.ts [hunk 1]
```diff
@@ -19,12 +19,6 @@ enum Output {
     ALL = 0x0f
 }
 
-enum OutputType {
```
Comment (jwunderl) [inline libs/core/output.ts:22]: Please do not include unrelated changes in your prs. In this case, it's possible a user project is referring to this for some reason so removing it would break things; if there's no particular reason to remove it, it's better to just leave it in.

**Review by jwunderl — CHANGES_REQUESTED** [review]

File: libs/core/output.ts (hunk 1 above)
Comment (THEb0nny) [reply libs/core/output.ts]: Ok i got it back

**Review by jwunderl — APPROVED** [review]
